### PR TITLE
Make 12.4.2 of chef

### PR DESCRIPTION
**Pull request by jkeiser**

### CODE REVIEW

File: config/projects/chefdk.rb [hunk 1]
```diff
@@ -32,8 +32,8 @@
 end
 
 # Uncomment to pin the chef version
-override :chef,           version: "stable"
-override :ohai,           version: 'stable'
+override :chef,           version: "12.4-stable"
```
Comment (schisamo) [inline config/projects/chefdk.rb]: Should a ChefDK change be in this PR?

Comment (jkeiser) [inline config/projects/chefdk.rb]: I'd like to build both chef and chef-dk, yeah--lets us get coverage of this new build on delivery.

Comment (ksubrama) [inline config/projects/chefdk.rb]: Why not just change what stable points to?

Comment (jkeiser) [inline config/projects/chefdk.rb]: I can fix the chef one that way, sure, but not ohai.

Comment (jkeiser) [inline config/projects/chefdk.rb]: Fixed chef to point to stable.

**Comment by ksubrama** [comment]
:+1:

**Pull request merged**